release.sh

```diff
@@ -0,0 +1,289 @@
+#!/bin/bash
+
+usage() {
+    cat <<EOF
+Usage: $0 [-h]
+
+This is the release script for Membrane RTC Engine.
+Run \`$0\`, then follow the instructions.
+
+Available options:
+    -h    Print this help and exit
+EOF
+}
+
+## Global variables
+# The order of these repos is important: Engine must be released first, then WebRTC, then the rest
+REPOS="engine webrtc hls rtsp file sip"
+
+# How many stages the release consists of (how many times does this script have to be run)
+MAX_STAGE="3"
```

Comment:
Why do we need that?

Reply:
It deletes the RELEASE_STAGE file after the release is done